### DISCUSSION TITLE
feat: GPT Crawler RAG pipeline

### DISCUSSION THREAD

**Discussion by zd1990** [open-webui/pipelines#195] [Ideas]
Thank you for your work and hope to integrate GPT Crawler into RAG's functionality

**Comment by justinh-rahb** [comment]
Do you mean RAG from web URLs? If so that's already implemented, if you type a # followed by a URL and then click it, it'll be added as a RAG "document" to your chat.

**Comment by sahajamit** [comment]
But it only scraps the content of given page URL. Is it possible to scan all the links on the page which takes to other section of the website?

**Comment by justinh-rahb** [comment]
https://github.com/mendableai/firecrawl

I've tested this, it works.
